src/sage/sets/condition_set.py

```diff
@@ -419,16 +428,17 @@ def _sympy_(self):
             sage: (5, 7, 9) in ST
             False
 
-            sage: Interval = ConditionSet(RR, x >= -7, x <= 4, vars=[x]); Interval
+            sage: Interval = ConditionSet(RR, x >= -7, x <= 4, vars=[x]); Interval      # needs sage.symbolic
```

Comment:
This raises a doctest warning
```
sage -t --long --warn-long 33.4 --random-seed=204097948585721018066760244858101334890 src/sage/sets/condition_set.py
**********************************************************************
File "src/sage/sets/condition_set.py", line 431, in sage.sets.condition_set.?._sympy_
Warning: Variable 'x' referenced here was set only in doctest marked '# needs sage.modules sage.symbolic'
    Interval = ConditionSet(RR, x >= -7, x <= 4, vars=[x]); Interval      # needs sage.symbolic
    [96 tests, 0.88 s]
```
My suggestion is to remove the empty line above.